data/hai/hai reveal 4 middle.txt

```diff
@@ -1391,7 +1397,37 @@ mission "Hai Reveal [A10] Demanding Satisfaction"
 			choice
 				`	(Walk to her.)`
 			`	"Captain <first> <last> led great fleets during a civil war of the humans. We will have the honor of fighting alongside this warrior of the humans."`
-			`	He looks at you appraisingly. "That is well, but there is a matter yet to be addressed. We cannot just 'waltz' through Hai and human space without being blasted. How will you convince your Elders to allow us passage?"`
+			branch nobody
+				not "Wanderers: Defend Vara Ke'sok: done"
+			`	"Indeed, we have had the honor of fighting against this warrior of humans. Quite a troublemaker on the battlefield." He looks at you. "Your professor friend was chatty before you left, and it's been easy to track you since then. We've been studying your actions for quite a while. Why are you here now, as an ally of this True Hai?"`
+			choice
+				`	"You and I have crossed swords before, but this is a matter that concerns us both: the safety and honor of the Hai."`
+					goto honor
+				`	"I fight for money and prestige. The Hai are paying plenty of both for protection against a pirate threat."`
+					goto mercenary
+				`	"I assumed furballs like you were too brain-dead to count to two."`
+			`	Teeneep rolls her eyes and makes no attempt to intervene as General Choot'k moves rapidly to cut your head in twain.`
+				die
+			label honor
+			`	He stares intensely at you with a piercing gaze. After a long moment, General Choot'k says, "You have proven an honorable warrior, Captain <last>, even if you have chosen the wrong side of one war. Yes, I am willing to fight alongside you, defending the Hai.`
+				goto spy
+			label mercenary
+			`	"If you prove a skilled and honorable warrior as you fight alongside us, I may consider paying you far higher than the Wanderers to help us take back our stolen worlds.`
+			label spy
+			`	"Besides," he says, patting you on the shoulder, "there's no better way to learn the tactics of the enemy than observing those of someone from their inner circle. You've been an invaluable source of information these past few months, but now I'll study you in person. You will teach me much about how to take our worlds back, Captain <last>.`
+			choice
+				`	"As I shall learn from you, General Choot'k."`
+				`	"You'll find human tactics to be quite advanced, General."`
+			action
+				# Hostilities cease until Captain Last fights the Unfettered again
+				"reputation: Hai (Unfettered)" >?= 1
```

Comment:
```suggestion
				"reputation: Hai (Unfettered)" >?= 0
```
And the other ones too, just so it’s a bit less easy to fight them and have them like you still.

Reply:
The standard throughout the code is to use "1" if they barely like you.